app/views/posts/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Posts</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Post</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @posts.each do |post| %>
+      <tr>
+      <% if user_signed_in? %>
+        <td><%= post.user.name %></td>
+      <%end%>
+        <td><%= post.post %></td>
+        <td><%# link_to 'Show', post %></td>
+        <td><%# link_to 'Edit', edit_post_path(post) %></td>
+        <td><%# link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' } %></td>
```

Comment:
## Commented out code
- [ ] Comments: It’s better to delete these **commented pieces of code**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))